tests/utils/kudo.go

```diff
@@ -149,6 +149,19 @@ func (c *KubernetesTestClient) LogObjectsOfKinds(namespace string, components []
 	}
 }
 
+func (c *KubernetesTestClient) PrintLogsOfPod(podName, namespace string) {
```

Comment:
this will fail when the pod has more then 1 container, lets pass the container name